notifications/log.go

```diff
@@ -0,0 +1,26 @@
+package notifications
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// Subsystem defines the sub system name of this package.
+const Subsystem = "NTF"
```

Comment:
I propose to make it 4-letter or 5-letter not to be confused with log levels (ERR/WRN/DBG). E.g. NOTIF or NTFC.